contribution.md

```diff
@@ -0,0 +1,59 @@
+Project;
+ Current name: KUWINDA(Trademark Pending)
+ Kind: FullStack Application(built using next.js)
+ Creator: E-man
+ Contributors: Ladu-man, Kon-AK,
+ Executive Supervisor: COCO.
+ Main gitHub branch: https://github.com/Emmanuel-Melon/kuwinda.git 
+ Database: SQL database provided by planetscale
+ 
+Frameworks used in project;
+ next.js(Entire App)
+ react.js(Front-End)
+
+Languages used; 
+ Typescript (97%)
+ Javascript (3%)
+
+Coding Format used;
+
+
+Database(SQL database);
+ Database located on planet scale at https://planetscale.com/
+
+Cloning the project;
+ Using Terminal
+  Go to terminal and insert command into your desired destination.;
+   gh repo clone Emmanuel-Melon/kuwinda
+ 
+ Using Code Editor
+  Clone from : https://github.com/Emmanuel-Melon/kuwinda.git
+
+Setting up the project;
+ Make sure you have a code editor installed.(I recommend VS code)
+ Make sure you have node.js installed
```

Comment:
Specify the `Node.js` version.
- The project has a `.nvmrc` file. It allows you to switch to the desired version via running `nvm use`. 
- Try including instructions on how to install `nvm` on both Mac and Windows.